src/MergePlugin.php

```diff
@@ -87,6 +87,11 @@ class MergePlugin implements PluginInterface, EventSubscriberInterface
     const PACKAGE_NAME = 'wikimedia/composer-merge-plugin';
 
     /**
+     * Name of the composer 1.1 init event.
+     */
+    const INIT = 'init';
```

Comment:
I think a name like `COMPAT_PLUGINEVENTS_INIT` might be a bit nicer to future maintainers.